code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -316,14 +300,18 @@
 /obj/item/robot_module/security
```

Comment:
since you've recoded them to not have their distinctive tools/weapons, could you enable secborgs for the ball?